### ISSUE TITLE
Cannot run Directus Suite

### ISSUE THREAD

**Issue by fnicastri** [directus/app#1153]
I installed Directus Suite from the latest build in a fresh container with Apache 2.4+ and php 7.1+.
But after the log-in I get only a loading spinner and the following errors in the console:
```
GET http://d7.test/_/settings 404 (Not Found)
GET http://d7.test/interfaces 404 (Not Found)
GET http://d7.test/server/ping 404 (Not Found)
GET http://d7.test/_/collections 404 (Not Found)
GET http://d7.test/_/users?fields%5B0%5D=id&fields%5B1%5D=first_name&fields%5B2%5D=last_name&fields%5B3%5D=title&fields%5B4%5D=status&fields%5B5%5D=timezone&fields%5B6%5D=roles.%2A&fields%5B7%5D=avatar.%2A&fields%5B8%5D=company 404 (Not Found)
GET http://d7.test/pages 404 (Not Found)
GET http://d7.test/_/collection_presets?filter%5Btitle%5D%5Bnnull%5D=1&filter%5Buser%5D%5Bnull%5D=1 404 (Not Found)
GET http://d7.test/_/relations?limit=-1 404 (Not Found)
```
The Response from the XHR calls is a 404 with this text (markup omitted):
```
The requested URL /_/collection was not found on this server.
```
And if I try to access the url directly i get:
```
{"error":{"code":300,"message":"Reading items from \"directus_collections\" collection was denied","class":"Directus\\Permissions\\Exception\\ForbiddenCollectionReadException","file":"\/var\/www\/wordpress\/src\/core\/Directus\/Permissions\/Acl.php","line":988}}
```

Thanks,
Francesco

**Comment by rw3iss** [comment]
> @rw3iss I'll try to include this in Friday's release 👍

Would be much appreciated, thanks @rijkvanzanten !

**Comment by rijkvanzanten** [comment]
@rw3iss Could you verify your MAMP is running in GCI or Module mode? I remember there being an issue where the Authorization header wouldn't be passed on. That would fit this description 1-1 as the app allows you to login, but all subsequent requests fail with a 401 because the API never receives the access token.

**Comment by rijkvanzanten** [comment]
I think this issue might be a duplicate of https://github.com/directus/api/issues/351

**Comment by rw3iss** [comment]
@rijkvanzanten Thanks. I think it might be running CGI mode already... only see one option in MAMP under 'PHP' settings to choose 'Individual PHP version for every host (CGI Mode)', which I have enabled.
I'll play around with it a little more if you think it's an environment issue (very well may be), maybe install Apache standalone or something.

**Comment by rijkvanzanten** [comment]
I'm worried that it's the same missing Authorization header problem again :/ I'd like to avoid having to put the access token in the query params as that would open up some more security risks.. (gets logged easier / more visible etc)

**Comment by rijkvanzanten** [comment]
Yeah the application is sending requests to the API (just like any other API consumer)

for example

`/_/items/movies`

It would send an additional HTTP header `Authorization: Bearer <token>`

However, it seems like certain servers combined with Apache (like MAMP in the wrong php mode) will strip out the Authorization header for some reason...

The alternative is that we start putting the token inside the URL itself `/_/items/movies?access_token=<token>`, but that means that it's more visible and is getting logged quicker.

**Comment by rw3iss** [comment]
K, I'll try an see if that's what's happening here in a bit. Thank you.

**Comment by rw3iss** [comment]
@rijkvanzanten In this case, since the frontend App is talking directly to API endpoints, shouldn't I be able to see the Authorization header being sent to the API in the network request from the browser? I don't see such header being sent (and don't think the backend environment issues matter if no header is being sent in the first place), no? Sorry, overlooked the request URL that was pointed directly at the API for /authenticate, thought it was talking to the App dev server, which was proxying to the API server. I would just like to confirm that the Authorization header should be in the frontend App network request headers being sent to the API. Should I not see it being sent there, even if the API environment might be stripping it thereafter?

**Comment by rijkvanzanten** [comment]
@rw3iss You should be seeing the authorization header in the request devtools in the browser: 

<img width="1234" alt="image" src="https://user-images.githubusercontent.com/9141017/49315938-49442880-f4bd-11e8-8142-0793f7f1bb18.png">

**Comment by rw3iss** [comment]
@rijkvanzanten Thanks. That's what I figured/expected... but it's not there, however, I'm confused because this request is coming directly from the Directus App in the browser, and the App (not API) is running under its own server (npm run dev), so there is no Apache there... but still no header being sent from browser/app. The backend Apache server for the API wouldn't strip it away if it was never sent in the first place... so I'm wondering why the browser wouldn't send this header to begin with... ?

**Comment by rijkvanzanten** [comment]
Do you have a running example of an app that doesn't send the headers? I just tried on a fresh install on a DigitalOcean droplet and on the live demo servers and it's sending the header there both as well..

**Comment by rijkvanzanten** [comment]
> so I'm wondering why the browser wouldn't send this header to begin with

I can't imagine that it would be stripped out at the browser side, but hey, this is webdevelopment so anything can happen really

**Comment by rijkvanzanten** [comment]
Oh, also `/authenticate` _doesn't_ have the authorization header, as you're not yet authorized (eg the app doesn't have the token yet).

**Comment by rijkvanzanten** [comment]
@fnicastri We just reproduced and solved the OP 500 issue by installing `php_curl` again `sudo apt-get install php-curl` and restarting apache `sudo systemctl restart apache2`. Could you verify php_curl is installed on your server?

**Comment by rijkvanzanten** [comment]
@rw3iss I'm still thinking that the PHP mode in your MAMP is the culprit. We've had other people using MAMP have this exact problem where MAMP would strip out the Authorization header for no apparent reason. Switching the MAMP php mode did the trick (I don't use MAMP so I'm not entirely sure what the name is)

**Comment by fnicastri** [comment]
@rijkvanzanten I think it is installed but I'm out of town without my dev laptop (🎉) and can't check.
I will check it after the 9th!
